WireAPI/.swiftpm/xcode/xcshareddata/xcschemes/WireAPITests.xcscheme

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is a separate scheme for the tests really required? Can't the tests be part of `WireAPI.xcscheme`?

Reply:
Unfortunately I do not know how to achieve this, would you be able to give me some hints?